src/Model/Photo.php

```diff
@@ -148,22 +148,31 @@ public static function getPhoto($resourceid, $scale = 0)
 	 * @brief Check if photo with given resource id exists
 	 *
 	 * @param string  $resourceid  Resource ID of the photo
+	 * @param array   $conditions  Array of extra conditions. Optional
 	 *
 	 * @return boolean
 	 */
-	public static function exists($resourceid)
+	public static function exists($resourceid, array $conditions = [])
```

Comment:
I would prefer using the "resource-id" just as another parameter in the conditions.